homeassistant/components/lcn/climate.py

```diff
@@ -0,0 +1,139 @@
+"""Support for LCN climate control."""
+
+from homeassistant.components.climate import ClimateDevice, const
+from homeassistant.const import (
+    ATTR_TEMPERATURE, CONF_ADDRESS, CONF_UNIT_OF_MEASUREMENT)
+
+from . import LcnDevice, get_connection
+from .const import (
+    CONF_CONNECTIONS, CONF_LOCKABLE, CONF_MAX_TEMP, CONF_MIN_TEMP,
+    CONF_SETPOINT, CONF_SOURCE, DATA_LCN)
+
+DEPENDENCIES = ['lcn']
```

Comment:
We don't need this anymore after introducing manifest.json.

Reply:
For the other platformas I fixed this in #23321